presto-prometheus/src/test/java/com/facebook/presto/plugin/prometheus/TestPrometheusConnectorConfig.java

```diff
@@ -69,7 +69,7 @@ public void testFailOnDurationLessThanQueryChunkConfig()
             throws Exception
     {
         PrometheusConnectorConfig config = new PrometheusConnectorConfig();
-        config.setPrometheusURI(new URI("http://doesnotmatter.com"));
+        config.setPrometheusURI(new URI("http://doesnotmatter.example.com:"));
```

Comment:
Yes, if you don't own a domain you shouldn't use it. example is allowed per DNS RFCs.

`doesnotmatter.com` is a real domain backed by real web servers and people shouldn't compose URLs that might go to things they don't control.

There are a number of examples of various major vendors (and individuals) messing up the internet in scary ways because they included strings and didn't realize that users might actually use the things. (A number of the examples are from Microsoft, but plenty aren't.)